docs/user-guide/node-api.md

```diff
@@ -46,6 +46,10 @@ For more detail usage, see [Globby Guide](https://github.com/sindresorhus/globby
 
 `stylelint.lint()` returns a Promise that resolves with an object containing the following properties:
 
+### `code`
+
+A string that contains the autofixed code, if the `fix` option is set to `true` and the `code` option is provided. Otherwise, it is `undefined`.
```

Comment:
question

If the rule is not fixable—e.g. `color-no-invalid-hex`—Id expect to get the same code back and not `undefined`.
Am I wrong?

Reply:
I believe it refers to the `fix` option rather than the fixability of individual rules. @haocheng6 Is that right? 

If so, the same code will return in `code` if the `fix` option is used on a config containing only unfixable rules like `color-no-invalid-hex`. 

`undefined` is used when the `fix` option isn't set, which feels intuitive as Stylelint will not attempt to transform the inputted code.

Reply:
@jeddy3 In this doc, I think the `fix` option refers to the `fix` option for the Node.js API or the `fix` option in the config file.

@Mouvedia @jeddy3 Both of you are correct. I have added two new test cases in 5b1c778686959eb929807d85628ac8a73f7f7235. Hopefully, the new test cases can help us ensure the `standalone` function is working as expected.